ui/app/templates/clients/client/index.hbs

```diff
@@ -793,7 +793,7 @@
               {{capitalize a.item.name}}
               Attributes
             </div>
-            {{#if a.item.attributes.structured}}
+            {{#if a.item.attributesShort}}
```

Comment:
The conditional here now matches the property we use, rather than being a proxy for it.